src/components/Navigation.tsx

```diff
@@ -110,8 +110,9 @@ const Navigation = (props: Props) => {
         <Navbar height='100vh' width={{ sm: 300 }} p="md" fixed>
           <Navbar.Section grow>
             <Group className={classes.header} position="apart">
-              <Text>Ad4min</Text>
-              <Code>v0.0.3</Code>
+              <Image src="Logo44.png"></Image>
+              <Text>AD4Min</Text>
+              <Code>v0.0.6-alpha</Code>
```

Comment:
This can be a variable read from package.json with `import {version} from './package.json';`

Reply:
👍